### ISSUE TITLE
opt.scale is always ignored

### ISSUE THREAD

**Issue by donmccurdy** [uber-common/viewport-mercator-project#24]
If `scale` is provided, its value is ignored and replaced with a scale computed from the default `zoom`.

Relevant lines: https://github.com/uber-common/viewport-mercator-project/blob/master/src/perspective-mercator-viewport.js#L76-L87

**Comment by ibgreen** [comment]
> If scale is provided, its value is ignored and replaced with a scale 

@donmccurdy Yes, the idea is that the class uses the named parameters listed in the constructor. It's intended to be a typical set of web mercator parameters, such as used by mapbox-gl/react-map-gl/deck.gl.

If you have `scale` instead of `zoom`, just convert it to zoom before you pass it in by calculating the base 2 logarithm:
```
new WebMercatorViewport({
   ...,
   zoom: Math.log2(scale)
});
```

**Comment by ibgreen** [comment]
@donmccurdy Just noticed that the docs do list `scale`. I'll take a second look to decide what should be changed. In the meantime, use the workaround above.
